beacon-chain/operations/attestations/kv/aggregated.go

```diff
@@ -13,15 +13,34 @@ func (p *AttCaches) SaveAggregatedAttestation(att *ethpb.Attestation) error {
 	if !helpers.IsAggregated(att) {
 		return errors.New("attestation is not aggregated")
 	}
-
-	r, err := ssz.HashTreeRoot(att)
+	r, err := ssz.HashTreeRoot(att.Data)
 	if err != nil {
 		return errors.Wrap(err, "could not tree hash attestation")
 	}
 
+	key := string(r[:])
+	var atts []*ethpb.Attestation
+	d, ok := p.aggregatedAtt.Get(key)
```

Comment:
doing `p.aggregatedAtt.Get(string(r[:]))` is actually faster underneath the hood in Go: https://github.com/golang/go/issues/3512, at least for maps (small tidbit)